scripts/rspec/rspec.ps1

```diff
@@ -76,9 +85,14 @@ function GenerateRuleClassesCS() {
     $FilesMap = @{
         "Rule.CS.cs"     = "${RulesFolderCS}\\${className}.cs"
         "Test.CS.cs"     = "${RulesFolderTests}\\${className}Test.cs"
-        "TestCase.CS.cs" = "${TestCasesFolder}\\${className}.cs"
     }
-    WriteClasses $FilesMap $ClassName
+
+    if (-Not (Test-Path -Path "${TestCasesFolder}\\${className}.cs" -PathType Leaf))
+    {
+        $FilesMap["TestCase.CS.cs"] = "${TestCasesFolder}\\${className}.cs"
```

Comment:
Super nitpick (optional): I would have renamed `className` into `ClassName` on the whole function, for consistency with GenerateRuleClassesVB